### ISSUE TITLE
Is it normal for accuracy to rapidly decrease to worse than random before increasing?

### ISSUE THREAD

**Issue by brodie-brodes** [4uiiurz1/keras-arcface#13]
I've integrated the Arcface loss layer as the output of a ResNet50 binary classification network. However, the training accuracy rapidly drops to ~3% despite there only being two classes. Meanwhile, the loss steadily decreases as normal. Then around epoch 10, the accuracy shoots up to around 89% very rapidly.

Is it normal for training to follow these patterns when using Arcface? I perhaps haven't understood the loss function well enough.

**Comment by ElisonSherton** [comment]
Hey @brodie-brodes, did you figure out why this issue happened? It is really interesting and I have also observed this issue.
